### PR TITLE
Configure number of workers in ES/LS outputer

### PR DESCRIPTION
**Pull request by urso**
configure number of workers per Host to be used by loadbalancing mode via
'worker' option

### CODE REVIEW

**Comment by ruflin** [comment]
Do we have any recommendations for the number of workers? 1 worker per elasticsearch node? Needs rebasing again.

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -291,6 +291,10 @@ output:
 In the previous example, the Elasticsearch nodes are available at https://10.45.3.2:9220/elasticsearch and
 https://10.45.3.1:9230/elasticsearch.
 
+===== worker
+
+Number of workers per configured host publishing events to Elasticsearch. Best
```
Comment (ruflin) [inline docs/configuration.asciidoc]: I suggest to make this doc a little bit more extensive to better explain what a worker here exactly is.

Comment (urso) [inline docs/configuration.asciidoc]: suggestions? It's just number of go-routines publishing to elasticsearch.

Comment (urso) [inline docs/configuration.asciidoc]: See logstash doc: "The number of workers to use for this output" :)

Comment (ruflin) [inline docs/configuration.asciidoc]: Worker defines the number of processes started to publish data to each host defined under hosts.

Comment (urso) [inline docs/configuration.asciidoc]: Mmmhhh... this sounds like total number of workers for all hosts. Like if workers = 2 and I have 4 hosts only 2 processes are started. But it is per host, so 8 processes will be started.

Comment (ruflin) [inline docs/configuration.asciidoc]: @urso That is a good explanation. It should be added as an example.

Comment (ruflin) [inline docs/configuration.asciidoc]: @urso Following suggestion for the text: 

```
Number of workers per configured host publishing events to Elasticsearch. This is best used with load balancing mode enabled. Example: If you have 2 hosts and 3 workers, in total 6 workers are started (3 for each host).
```

Comment (urso) [inline docs/configuration.asciidoc]: Like it.

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -466,6 +471,12 @@ List of known logstash servers to connect to. All entries in this list can
 contain a port number. If no port number is given, the port options value is
 used as default port number.
 
+===== worker
+
+Number of workers per configured host publishing events to Elasticsearch. This
```
Comment (ruflin) [inline docs/configuration.asciidoc]: Copy paste: Should be Logstash

**Pull request merged**